.github/workflows/evm-test.yml

```
@@ -0,0 +1,70 @@
+      run:
+        working-directory: bridge-evm
+    name: Foundry project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+            version: stable
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)